src/pages/home/base/mobile-core/identity/index.md

```diff
@@ -14,6 +14,18 @@ import Tabs from './tabs/index.md'
 
 The Identity extension enables your app with the Experience Cloud ID (ECID). This service helps with the synchronization of Adobe and other customer identifiers.
 
+Include the Identity extension in your application when using any of the following Adobe Solution extensions:
+
+   * [Adobe Analytics](../../../../solution/adobe-analytics/index.md)
+   * [Adobe Audience Manager](../../../../solution/adobe-audience-manager/index.md)
+   * [Adobe Campaign Standard](../../../../solution/adobe-campaign-standard/index.md)
+   * [Adobe Media Analytics](../../../../solution/adobe-media-analytics/index.md)
+   * [Adobe Target](../../../../solution/adobe-target/index.md)
+
+When using 3rd party extensions, check with the extension developer to understand if the Identity extension is a required dependency.
```

Comment:
Minor nit - but generally for our docs we write it as "third party", rather than "3rd party".